lib/prm/employees/employee_search.ex

```diff
@@ -7,6 +7,8 @@ defmodule PRM.Employees.EmployeeSearch do
     field :party_id, Ecto.UUID
     field :division_id, Ecto.UUID
     field :legal_entity_id, Ecto.UUID
+    field :tax_id, :string
+    field :edrpou, :string
```

Comment:
it's better to use embed schema, when you need to search in related tables.
See [here](https://github.com/Nebo15/prm.api/blob/master/lib/prm/entities/legal_entity_search.ex#L14) and [here](https://github.com/Nebo15/prm.api/blob/master/lib/prm/entities/entities.ex#L84)